dhis-2/dhis-api/src/main/java/org/hisp/dhis/relationship/RelationshipService.java

```diff
@@ -74,6 +74,15 @@
      */
     Relationship getRelationship( long id );
 
+    /**
+     * Checks for the existence of a relationship by UID. Takes into account
+     * also the deleted relationships.
+     *
+     * @param uid Relationship UID to check for.
+     * @return relationship exists or not.
+     */
+    boolean relationshipExistsIncludingDeleted( String uid );
```

Comment:
do we have any convention for methods with boolean return types like this one
like?

`hasRelationship`
`isRelationship`

Reply:
Not really. Even less for the include deleted ones